### ISSUE TITLE
The cursor position go to head of first line when I press enter key on Microsoft Edge

### ISSUE THREAD

**Issue by GoebelL** [servicecatalog/development#128]
[Version] 
16.0.5

[Client]
Microsoft Edge 25.10586.0.0
Microsoft EdgeHtml 13.10586

[Frequency]
Always

[How to reproduce]
1. Login as supplier using Microsoft Edge
3. Go to Update service
4. Click description text field
5. Move to end of line
6. Press enter key

[Expected result]
Cursor go to next line

[Observed result]
Cursor go to head of first line

[Note]
This issue occurred in all of Rich Text Editor.

**Comment by grubskim** [comment]
[Details of fix]
Added enter key handling to the ckeditor setup. It now enters the <br>.

**Comment by SylviaDehn** [comment]
[Build] 2016/08/17 reopen
Rich text editor works differently for all browsers now.

**Comment by StavrevaS** [comment]
The following ckeditor settings solve the problem to have the same behavior as in other browsers:
- shiftEnterMode: CKEDITOR.ENTER_BR
- enterMode: CKEDITOR.ENTER_P
- autoParagraph: false

**Comment by SylviaDehn** [comment]
[Build] 2016/08/17 reopen.
Original Problem is not fixed.

**Comment by grubskim** [comment]
I have tested again three cases: before my commit 025f154 (case A), after my commit (case B), and after commit 7f6203b (case C). 

Browsers used: IE9, 10, 11, Firefox, Chrome, Edge.

1) In cases A and C enter does not work in Edge.
2) Editor does not display properly at all in all three cases in IE10 (components flashing, displaying partially, or not displaying)
3) In case A after pressing enter:
In IE11 it enters br tag, closed br tag, encloses them with p tag.
In IE9 it enters closed p tag
In Chrome it enters br tag enclosed in p tags
In Firefox it enters br type="_moz" tag enclosed in p tags
4) In case B enter key works in Edge and enters a br tag
In Chrome and FF br tags as well
IE 9 br tag
IE 11 br and closed br tag

The bottom line is that, in my opinion, the most consistent behaviour is in option B. Also it works in Edge, and there is no visible difference between br and br enclosed in p.

Please consider this and advise if we can use the br tags, or we should stick to the old, not working with Edge version, or invest more time to fix it in a different way (what is expected?).

**Comment by DawidCh** [comment]
@GoebelL can you advise Michał what to do?
